rendermodules/mesh_lens_correction/schemas.py

```diff
@@ -73,10 +73,25 @@ class MeshLensCorrectionSchema(PointMatchOpenCVParameters):
     z_index = Int(
         required=True,
         description="z value for the lens correction data in stack")
+    nfeature_limit = Int(
```

Comment:
These should probably be removed as they are now defined in the opencv pointmatch module.